packages/fxa-auth-server/lib/senders/templates/downloadSubscription.txt

```diff
@@ -1,9 +1,9 @@
 {{{subject}}}
 
-If you haven't already downloaded {{{product}}}, let's get started using all the features included in your subscription:
+{{{t "If you haven't already downloaded %(productName)s, let's get started using all the features included in your subscription:" }}}
 
 {{{link}}}
 
-Questions about your subscription? Our support team is here to help you:
+{{{t "Questions about your subscription? Our support team is here to help you:" }}}
```

Comment:
I'm seeing a handful of unescaped strings `{{{`. Can we switch to escaping `{{` unless necessary?

Reply:
This is a text format email - it needs to be unescaped. We also have tests that fail if escaped characters appear in text format output